client/web/src/enterprise/batches/list/BatchChangeListPage.tsx

```diff
@@ -192,7 +192,7 @@ export const BatchChangeListPage: React.FunctionComponent<React.PropsWithChildre
             )}
             {selectedTab === 'batchChanges' && (
                 <>
-                    <BatchChangeStatsBar className="mb-4" />
+                    <BatchChangeStatsBar className="mb-4" namespaceID={namespaceID} />
```

Comment:
```suggestion
                    {!namespaceID && <BatchChangeStatsBar className="mb-4" />}
```

See comment further down, this would get around that problem.